TLSharp.Tests/TLSharpTests.cs

```diff
@@ -129,7 +128,7 @@ public virtual async Task AuthUser()
             var hash = await client.SendCodeRequestAsync(NumberToAuthenticate);
             var code = CodeToAuthenticate; // you can change code in debugger too
 
-            if (String.IsNullOrWhiteSpace(code))
+            if (string.IsNullOrWhiteSpace(code))
```

Comment:
no need for this cosmetic change, please undo (or separate this change into a different commit or PR)